### DISCUSSION TITLE
AutoGrade Error Github Classroom

### DISCUSSION THREAD

**Discussion by mbellis13** [community/community#17754] [GitHub Education]
I am trying github classroom for the first time and I am getting this error from all test students.  It says the tests pass, but there there is this resource not available error and on the dashboard it shows all students as having failed.  Any idea why the error?  Can't seem to find any good answers online

```
📝 test 1


  
  hello, ellis
✅ test 1


::***::

All tests passed

✨🌟💖💎🦄💎💖🌟✨🌟💖💎🦄💎💖🌟✨

Points 5/5
Resource not accessible by integration
Error: Autograding failure: HttpError: Resource not accessible by integration
```

**Comment by mbellis13** [comment]
anything on this?

**Reply by JessRudder** [reply]
Hello! We believe we've pushed a fix that should get the student repos back into a good state. Could you please let me know if you're still seeing the same issue?

**Comment by VictorNorman** [comment]
Please, github people, help!

**Reply by JessRudder** [reply]
Hello! We believe we've pushed a fix that should get the student repos back into a good state. Could you please let me know if you're still seeing the same issue?

**Reply by VictorNorman** [reply]
Yes!  Thank you @JessRudder . I'm thrilled to find my stuff working this morning!
